app/org/sagebionetworks/bridge/dynamodb/DynamoInitializer.java

```diff
@@ -315,7 +311,9 @@ static void initTables(final List<TableDescription> tables) {
                 logger.info("Creating table " + table.getTableName());
                 DYNAMO.createTable(createTableRequest);
             } else {
-                compareSchema(table, existingTables.get(table.getTableName()));
+                final TableDescription existingTable = existingTables.get(table.getTableName());
+                compareSchema(table, existingTable);
+                updateThroughput(table,existingTable);
```

Comment:
I'm not sure we should do this. This means that if someone changes the throughput in Dynamo, when DynamoInitializer comes back, it'll revert those changes. It would be much safer to use DynamoThroughput only for table creation, and if we want to sync up our DDB tables with our code, there should be an explicit process to do that.